modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml

```diff
@@ -329,12 +344,15 @@ components:
           $ref: '#/components/schemas/NoticeStatus'
         updatedAt:
           type: string
-          description: the notice updated date/time.
+          description: >-
+            The exact moment of the latest update of the notice by either user
+            or system.
           format: date-time
           example: '2022-01-01T13:15:55.89Z'
         submittedAt:
           type: string
-          description: the notice submitted date/time.
+          description: The user performed the latest update of the notice (can be system).
```

Comment:
This description seems inconsistent with the name "submittedAt".

Reply:
You are right. The correct description should be: The notice submission date/time.